utility/src/main/java/org/oppia/android/util/namevalidator/NameValidator.kt

```diff
@@ -0,0 +1,39 @@
+package org.oppia.android.util.namevalidator
```

Comment:
Please have a look at the new android_library and suggest if deps [ ] field is to be added

Reply:
I suggest setting up Bazel locally per the wiki and trying to build the app locally (since there are other BUILD files that will need to be updated here).